src/expipe_plugin_cinpla/scripts/utils.py

```diff
@@ -297,12 +297,14 @@ def compute_and_set_unit_groups(sorting, recording):
             unit_groups = recording.get_channel_groups()[np.array(list(extremum_channel_indices.values()))]
             sorting.set_property("group", unit_groups)
         else:
-            unit_groups = sorting.get_property("group").astype("str")
+            unit_groups = sorting.get_property("group").astype("U20")
```

Comment:
@mariapfj this was the cause of the last issue. In case there are only merges/splits in Phy, all groups are `nan` so casting to string makes a 3-character object. When setting a value to `tetrode*`, it became `tet`...